GNUmakefile

```diff
@@ -0,0 +1,14 @@
+include $(GNUSTEP_MAKEFILES)/common.make
+
+BUNDLE_NAME = MIPSCPU
+MIPSCPU_OBJC_FILES = \
+	MIPSCPU/MIPSCPU.m \
+	MIPSCPU/MIPSCtx.m
+
+#NOWARN=-Wno-format -Wno-return-type -Wno-unused-value -Wno-unused-variable -Wno-self-assign
+MIPSCPU_OBJCFLAGS=$(NOWARN) \
+	-I${HOME}/GNUstep/Library/ApplicationSupport/Hopper/HopperSDK/include \
```

Comment:
I had a look at your installer and with regard to the Hopper SDK it doesn't seem to do anything special besides downloading, unzipping and symlinking it to your location. 
As this repository already provides it's own Hopper SDK I currently don't see the point to use a different one here.
Same is true for https://github.com/makigumo/RISCV/pull/1